server/src/main/java/io/druid/server/coordination/SegmentChangeRequestDrop.java

```diff
@@ -62,4 +62,29 @@ public String toString()
            "segment=" + segment +
            '}';
   }
+
+  @Override
+  public boolean equals(Object o)
+  {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
```

Comment:
Actually, I don't know what the behavior of this on instantiated classes is either.